_board/seeed_xiao_esp32s3.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: download
+board_id: "seeed_xiao_esp32s3"
```

Comment:
We don't currently have a board that has the id `seeed_xiao_esp32s3` in the circuitpython core repo yet. So this listing will not be able to properly list its firmware download

Reply:
Our ID is called is seeed_xiao_esp3_2s3_sense?

Reply:
Our ID is called seeed_esp32_xiao_s3_sense

Reply:
Our ID is called seeed_esp32_xiao_s3_sense?

Reply:
Our ID is called seeed_xiao_esp32_s3_sense?

Reply:
@Jasionf we need to figure out some details about the sense version PID and come to a consensus on how best to include the non-sense version. There are some active discussions around this in discord and over in the core repo under the PR where the sense version was added to the core.  Once we have a clear path forward I will post an update here with more info.

Reply:
We'd like to have this device added to the core first with a PR to https://github.com/adafruit/circuitpython that will require VID / PID codes which can be obtained from pid.codes. There is a page here: https://pid.codes/howto/ that details the process of submitting a PR to get codes allocated that way. 

When this device does get added to the core it will be under the board_id `seeed_xiao_esp32s3`, and then after that is merged in we finalize this one to get it added to the downloads page.

In the meantime, I think that the sense version of the firmware can work on the non-sense device. You'll just need to be wary of some of the pins not being easily accessible to hardware.

Reply:
Looks like some other peeps used to do a this: 
https://github.com/adafruit/circuitpython/issues/9507#issuecomment-2298119726
But I wasn't able to xiao_esp32s3 inside circuitpython, did I miss something?